polybar-scripts/info-optimus-manager/info-optimus-manager.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
lets use `/bin/sh` for this

Reply:
this has been fixed in a later commit and all the bash specific language has been removed :)